packages/grafana-data/src/types/config.ts

```diff
@@ -208,7 +208,6 @@ export interface GrafanaConfig {
   http2Enabled: boolean;
   dateFormats?: SystemDateFormatSettings;
   grafanaJavascriptAgent: GrafanaJavascriptAgentConfig;
-  customTheme?: any;
```

Comment:
When was this used previously? 🤔

Reply:
i cannot find (or know of) a use of it anywhere, hence why i removed it 😄